haystack/nodes/audio/_text_to_speech.py

```diff
@@ -33,17 +34,20 @@ def __init__(
         model_name_or_path: Union[str, Path],
         use_gpu: bool = True,
         transformers_params: Optional[Dict[str, Any]] = None,
+        devices: Optional[List[Union[str, torch.device]]] = None,
     ):
         """
         :param model_name_or_path: The text to speech model, for example `espnet/kan-bayashi_ljspeech_vits`.
         :param use_gpu: Whether to use GPU (if available). Defaults to True.
         :param transformers_params: Parameters to pass over to the `Text2Speech.from_pretrained()` call.
+        :param devices: List of GPU devices to limit inference to certain GPUs and not use all available ones (e.g. [torch.device('cuda:0')]).
```

Comment:
Does it mean the following:
List of GPU devices you want to limit inference to so as not to use all available GPUs. Example: [torch.device(cuda:0)].